themes/default/content/docs/intro/vs/terraform/_index.md

```diff
@@ -0,0 +1,170 @@
+---
```

Comment:
This page shows net-new because it was moved from above, to make room for the sibling "equivalents" page.